pss/pss.go

```diff
@@ -52,11 +59,41 @@ func (p *Pss) Send(ctx context.Context, targets [][]byte, topic trojan.Topic, pa
 		return nil, err
 	}
 
+	tag := chunk.NewTag(1, "pss-chunks-tag", 0, false)
+	tc.WithTagID(tag.Uid)
+
 	// SAVE trojanChunk to localstore, if it exists do nothing as it's already peristed
-	// TODO: for second phase, use tags --> listen for response of recipient, recipient offline
 	if _, err = p.localStore.Put(ctx, chunk.ModePutUpload, tc); err != nil {
 		return nil, err
 	}
+	tag.Inc(chunk.StateStored)
+	tag.Total = 1
+
+	monitor := &Monitor{
+		chunk:  tc,
+		tag:    tag,
+		states: make(map[chunk.State]string),
+	}
+
+	monitor.updateState()
```

Comment:
what about making this function a go routine? so that states are updated as tags are changed, rather than having to manually call this each time

Reply:
This could be a possible solution, but we would create one for each Monitor?
I'm not sure how to implement it, but it makes sense that it should async

Reply:
>  but we would create one for each Monitor?

yes, same as right now, only with `go`